src/app.js

```diff
@@ -46,4 +51,10 @@ app.use('/api/v1/classes', classesRouter);
 app.use(notFound);
 app.use(errorHandlerMiddleware);
 
-module.exports = app;
+const server = http.createServer(app);
+const io = new Server(server, {
+  cors: { origin: 'http://localhost:5173', method: ['GET', 'POST'] },
```

Comment:
Hi Maria! We already deployed our project, don't you mind to add the additional origin linked the app's url?

Reply:
Thank you, Anna!
✅Done